docs/fastdds/dds_layer/core/policy/policy.rst

```diff
@@ -1,5 +1,2259 @@
+
+This QoS Policy controls the mechanism used by the service to ensure that a particular entity on the network is still
+alive.
+There are different settings that allow distinguishing between applications where data is updated periodically and
+applications where data is changed sporadically.
+It also allows customizing the application regarding the kind of failures that should be detected by the liveliness
+mechanism.
+
+List of QoS Policy data members:
+
++--------------------------+-------------------------------+-------------------+
+| Data Member Name         | Type                          | Default Value     |
++==========================+===============================+===================+
+| kind                     | :ref:`livelinessqospolicykind`| AUTOMATIC         |
++--------------------------+-------------------------------+-------------------+
+| lease_duration           | fastrtps::Duration_t          | c_TimeInfinite    |
++--------------------------+-------------------------------+-------------------+
+| announcement_period      | fastrtps::Duration_t          | c_TimeInfinite    |
++--------------------------+-------------------------------+-------------------+
+
+* **Kind**: This data member establishes if the service needs to assert the liveliness automatically or if it needs
+  to wait until the liveliness is asserted by the publishing side.
+  See :ref:`livelinessqospolicykind` for further details.
+* **Lease Duration**: Amount of time to wait since the last time the DataWriter asserts its liveliness to consider
+  that it is no longer alive.
+* **Announcement Period**: Amount of time between consecutive liveliness messages sent by the DataWriter.
+  This data member only takes effect if the kind is AUTOMATIC or MANUAL_BY_PARTICIPANT and needs to be lower than the
+  `lease_duration`.
+
+.. note::
+     This QoS Policy concerns to Topic, DataReader and DataWriter entities.
+     :raw-html:`<br />`
+     It cannot be changed after the creation of the entities.
+
+.. warning::
+    For DataWriters and DataReaders to match, they must follow the compatibility rule. See :ref:`liveliness_compatibilityrule`
+    for further details.
+
+.. _livelinessqospolicykind:
+
+LivelinessQosPolicyKind
+"""""""""""""""""""""""
+
+There are three possible values:
+
+* **Automatic**: The service takes the responsibility for renewing the leases at the required rates, as long as the
+  local process where the participant is running and the link connecting it to remote participants exists, the
+  entities within the remote participant will be considered alive.
+  This kind is suitable for applications that only need to detect whether a remote application is still running.
+* The two **Manual** modes require that the application on the publishing side asserts the liveliness periodically
+  before the lease_duration timer expires. This action can be done by calling `assert_liveliness` member function, or
+  by writing some data.
```

Comment:
Clarifying a little bit more:
```suggestion
* The two **Manual** modes require that the application on the publishing side asserts the liveliness periodically
  before the lease_duration timer expires.
  Publishing any new data value implicitly asserts the DataWriter's liveliness, but it can be done explicitly
  by calling the `assert_liveliness` member function.
```